src/css/Header.css

```diff
@@ -1,23 +1,61 @@
-@import url("https://fonts.googleapis.com/css2?family=Quattrocento:wght@700&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Cinzel+Decorative:wght@900&display=swap");
 @import url("https://fonts.googleapis.com/css2?family=EB+Garamond:wght@450&family=Quattrocento:wght@700&display=swap");
 
-h1,
+#logo {
+  font-family: "Cinzel Decorative", cursive;
+  color: white;
+  font-size: 60px;
+}
+
+span {
+  font-family: "Cinzel Decorative", cursive;
+  color: white;
+}
+
+h1 {
+  font-family: "Quattrocento", serif;
+  color: white;
+  font-size: 40px;
+}
+
 h2,
-h3,
 h4,
 h5,
 h6 {
   font-family: "Quattrocento", serif;
```

Comment:
Did you replace this font with Cinzel Decorative? Then font-family should be updated for the other headings as well.